plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -513,7 +568,9 @@
     api_host: helldorado
     name: spynal
     node: sabrewulf
-    net: '{"net0":"virtio,bridge=vmbr1,rate=200", "net1":"e1000,bridge=vmbr2,"}'
+    net: 
+      net0: 'virtio,bridge=vmbr1,rate=200'
+      net1: 'e1000,bridge=vmbr2'
```

Comment:
```suggestion
      net1: 'e1000,bridge=vmbr2,'
```
The original version had a trailing comma. I have no idea whether that's totally irrelevant or very important :)

Reply:
It is totally unimportant and definitively ignored by `proxmox_kvm` :)